### ISSUE TITLE
Is real-time re-sampling technically possible?

### ISSUE THREAD

**Issue by wa1** [spotify/android-sdk#407]
## TL;DR 
Spotify removed the _Running_ feature from the Android app `:(`. It helped me enjoy and improve my running. I want to take a stab at rebuilding that functionality using the SDK and adjusting tempo, if that's technically possible.

## "Simple" version:
So, one way is to get the track properties of a user's songs and filter by `tempo`, in order to create running playlists with a certain BPM, instead of using Spotify's too wide-range BPM playlists. I think I have what I need to create a service like that, I'd like to create something smarter than that.

## Question..
I'm toying with the idea of doing *real-time re-sampling* of playback in order to change the tempo (and preferably adjusting pitch accordingly), would that even be technically possible using only the SDK?

**Comment by arirawr** [comment]
Hi @wa1 - no, you are not able to manipulate audio through the SDK.

I also recommend you take a look at our Developer Terms of Service you accept by using out developer tools: https://beta.developer.spotify.com/terms/
